src/utils/SkyblockUtils.js

```diff
@@ -179,6 +179,8 @@ function getTrophyFishRank(level) {
     return 'Gold';
   } else if (4 === level) {
     return 'Diamond';
+  } else if (5 === level) {
+    return 'Diamond';
```

Comment:
Make it so if level is grater then or equal to 4 it does Diamond

Reply:
@Kathund done